src/hooks/useTeamsBreadcrumbs.ts

```diff
@@ -0,0 +1,61 @@
+import { useMemo } from 'react';
+import { setTeamColor } from '~v5/common/TeamReputationSummary/utils';
+import useColonyContext from './useColonyContext';
+
+export const useGetSelectedTeamFilter = () => {
+  const { colony } = useColonyContext();
+  const { domains } = colony || {};
+
+  const urlParams = new URLSearchParams(window.location.search);
```

Comment:
I think it would be better to use https://reactrouter.com/en/main/hooks/use-search-params